view/templates/item/compose.tpl

```diff
@@ -103,4 +103,19 @@
 </div>
 <script>
 	dzFactory.setupDropzone('#dropzone-{{$id}}', 'comment-edit-text-{{$id}}');
+
+	document.addEventListener("DOMContentLoaded", function() {
+		var textareas = document.querySelectorAll(".expandable-textarea");
+
+	textareas.forEach(function(textarea) {
+		textarea.addEventListener("input", function() {
+			this.style.height = "auto";
+			this.style.height = (this.scrollHeight) + "px";
+		});
+
+		// Set initial height
+		textarea.style.height = "auto";
+		textarea.style.height = (textarea.scrollHeight) + "px";
+	});
+});
```

Comment:
Sorry for nitpicking, but this closing bracket doesn't align to the opening one, which make the code harder to read.

Reply:
I'm not sure if I've understood you correctly. I'm still practising at the moment and would be grateful for any tips